### ISSUE TITLE
How to train from scratch?

### ISSUE THREAD

**Issue by ghost** [clovaai/CRAFT-pytorch#159]
Hi, I understand you have reasons not to post the training code possibly for commercial reasons or whatever. I have several questions regarding the missing training code:

- Assuming I have word and character level annotations, and I prefer strong supervision, which training steps would be different than the ones proposed in the paper?
- Can you show an actual example image with region and affinity scores calculated?
- How to calculate a proper loss function? There is no explicit description of a loss function in the paper so it'd be great if you provide some details.
- Are there references you recommend to rely upon? except for this [one](https://github.com/backtime92/CRAFT-Reimplementation) because it just doesn't work.
- Does your implementation support training from absolute scratch (without pretrained weights)?
- Does your implementation support cursive characters?
